lib/utils/latest_version.rb

```diff
@@ -8,7 +8,8 @@ class LatestInSpecVersion
   # fetches the latest version from rubygems server
   def latest
     uri = URI('https://rubygems.org/api/v1/gems/inspec.json')
-    res = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https') {|http|
+    res = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https',
+                          open_timeout: 0.5) {|http|
       http.read_timeout = 0.5
```

Comment:
I agree with your comment that this should be moved up to the `#start` opts for consistency. Thank you for suggesting it!